plugin/src/test/java/io/jenkins/plugins/forensics/git/miner/GitMinerStepITest.java

```diff
@@ -43,6 +45,34 @@ public void shouldFillTableDynamically() {
                 .hasFileName("file").hasAuthorsSize(1).hasCommitsSize(1);
     }
 
+    /** Verifies that the mining process is incremental. */
+    @Test
+    public void shouldMineRepositoryIncrementally() throws IOException {
+        writeFileAsAuthorFoo("First");
+
+        FreeStyleProject job = createJobWithMiner();
+        Run<?, ?> build = buildSuccessfully(job);
+
+        getJenkins().assertLogContains("created report for 2 files", build);
+
+        build = buildSuccessfully(job);
+
+        getJenkins().assertLogContains("created report for 0 files", build);
```

Comment:
Add another commit and yet another build that detects one additional commit.

Reply:
Like this? 
``` 
        writeFileAsAuthorFoo("Second");

        build = buildSuccessfully(job);

        getJenkins().assertLogContains("created report for 2 files", build);

```
Or should I check with hasCommitsSize(), like in the other test?

Reply:
No that is just fine